### PR TITLE
Riskdev rob

### PR DESCRIPTION
**Pull request by ramarty**
- Update risk tab on dashboard, including where elements appear
- Add movement lines to risk map
- Make risk table formattable, to add colored bars
- Fix issue with password not working. For some reason this didn't work when splitting into different files (eg, server.R, ui.R), but worked when combining everything into app.R, so just kept everything in app.R
- Cleaned up & added additional comments in app.R, and made outline structure so a bit easier to find different code chunks.
- Added sparkline to table on right of main page, so user can see trends
- Updated readme, particularly describing smaller datasets that are used by the dashboard

### CODE REVIEW

File: dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_distance_traveled.R [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Clean Subscribers Data
+
+EXPORT <- T
+
+for(unit in c("district", "ward")){
+  for(metric in c("mean_distance", "stdev_distance")){
```
Comment (LeonardoViotti) [inline dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_distance_traveled.R:6]: Shame

File: dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data.R [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 EXPORT <- T
 
+
+unit <- "district"
```
Comment (LeonardoViotti) [inline dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data.R]:
```suggestion

```

File: dashboard-dataviz/zimbabwe_dashboard/app.R [hunk 3]
```diff
@@ -1,28 +1,1948 @@
...
+  setwd("C:/Users/wb521633/Documents/Github/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
   )
 }
 
 if (Sys.info()[["user"]] == "wb519128") {
-  setwd(
-    "C:/Users/wb519128/GitHub/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
+  setwd("C:/Users/wb519128/GitHub/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
   )
 }
```
Comment (LeonardoViotti) [inline dashboard-dataviz/zimbabwe_dashboard/app.R:23]: Did you move the dashboard back to just one file or did you branch before it was splited?

**Review by LeonardoViotti — COMMENTED** [review]
Looks good overall.
Was it a conscious decision to put all the code for the dashboard in the app.R file? If so, does this code reflect the published version of the dashboard?

File: dashboard-dataviz/zimbabwe_dashboard/app.R (hunk 3 above)
Comment (ramarty) [reply dashboard-dataviz/zimbabwe_dashboard/app.R:23]: I'm fairly sure I branched before I combined everything back into one

File: dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data.R (hunk 2 above)
Comment (ramarty) [reply dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data.R]: For some reason enabling the password wasn't working when things were split up into multiple files, but putting everything back into one file fixed it. Not sure why. Might have something to do with two separate "UIs" being defined (one for the password landing page, and one for the main page), but not exactly sure why things don't work when split up.

File: dashboard-dataviz/zimbabwe_dashboard/app.R (hunk 3 above)
Comment (ramarty) [reply dashboard-dataviz/zimbabwe_dashboard/app.R:23]: And yes, this reflects the published version

File: dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_distance_traveled.R (hunk 1 above)
Comment (ramarty) [reply dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_distance_traveled.R:6]: hehehe

**Pull request merged**